saleor/settings.py

```diff
@@ -320,7 +320,9 @@ def get_bool_from_env(name, default_value):
 
 # VAT configuration
 # Enabling vat requires valid vatlayer access key.
+# If you are subscribed to a paid vatlayer plan, you can enable HTTPS.
 VATLAYER_ACCESS_KEY = os.environ.get('VATLAYER_ACCESS_KEY')
+VATLAYER_USE_HTTPS = get_bool_from_env('VATLAYER_USE_HTTPS', False)
```

Comment:
Can we use `ENABLE_SSL` variable here?

Reply:
The settings `ENABLE_SSL` to set `VATLAYER_USE_HTTPS`? That would not work as the issue is that most of the people that will use the taxes feature won't have a paid vatlayer plan (thus no HTTPS available for API requests).

Reply:
I didnt know that paid plan is required for this, Thanks for the clarification